generic/ras.py

```diff
@@ -40,6 +40,14 @@ def setUp(self):
         architecture = os.uname()[4]
         if "ppc" not in architecture:
             self.skip("supported only on Power platform")
+        if 'PowerNV' in process.system_output('grep ^platform /proc/cpuinfo'):
+            nv_flag = False
+            for cmd in ['lsmcode', 'lsprop', 'nvram', 'ppc64_cpu']:
+                if cmd in self.name.name:
```

Comment:
If that's the case, `self.name.name` will contain a long string with path, module name, class name and method name, like `/home/apahim/avocado/tests/test.py:MyTest.test`. Very prone to break this check.
On the other hand, `self._testMethodName` will have only the method name.